### PR TITLE
make a new excpetion for dropping gaff support

### PR DESCRIPTION
**Pull request by mikemhenry**

### CODE REVIEW

**Comment by mikemhenry** [comment]
testing this here https://github.com/OpenFreeEnergy/openfe/pull/847

File: openmmforcefields/generators/system_generators.py [hunk 1]
```diff
@@ -260,10 +261,18 @@ def __init__(
                         template_generator_kwargs=self.template_generator_kwargs,
                     )
                     break
-                except (ValueError, NotImplementedError) as e:
+                except (ValueError, NotImplementedError, GAFFNotSupportedError) as e:
```
Comment (mattwthompson) [inline openmmforcefields/generators/system_generators.py:264]:
```suggestion
                except (ValueError, GAFFNotSupportedError) as e:
```

**Review by mattwthompson — APPROVED** [review]
LGTM % things working downstream

**Comment by mikemhenry** [comment]
It does!

**Pull request merged**